domain/src/main/java/no/difi/meldingsutveksling/domain/sbdh/StandardBusinessDocumentHeader.java

```diff
@@ -294,4 +294,16 @@ private Scope createDefaultScope() {
             return new Scope().setIdentifier(STANDARD_LEGACY);
         }
     }
+
+    public ZonedDateTime getExpectedResponseDateTime() {
+        Set<Scope> scope = this.getBusinessScope().getScope();
+        ZonedDateTime zonedDateTime = scope.stream().flatMap(s-> s.getScopeInformation()
```

Comment:
Kan returnere direkte her.

Reply:
Njeh, skal jo ikke gjøre noe mer med den. Blir jo en ekstra variabel for variabelens skyld, imo.